lib/Publisher/CalDAV/EventRealTimePlugin.php

```diff
@@ -385,7 +385,6 @@ function schedule(\Sabre\VObject\ITip\Message $iTipMessage) {
         ];
 
         $this->notifySubscribers($calendar->getSubscribers(), $dataMessage, $options);
-        $this->notifyInvites($calendar->getInvites(), $dataMessage, $options);
 
```

Comment:
Pro tip: 

 - [ ] Push https://github.com/linagora/twake-calendar-integration-tests/pull/135 branch directly onto the linagora repo
 - [ ] In run_test clone that branch in a `[NO_REVIEW]` commit

That way you have this code running onto the target PR.